### PR TITLE
Minor third-party bumps

### PR DESCRIPTION
**Pull request by NiLuJe**
* HB [2.9.1](https://github.com/harfbuzz/harfbuzz/releases/tag/2.9.1)
* KoboUSBMS [1.2.3](https://github.com/koreader/KoboUSBMS/releases/tag/v1.2.3)

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader-base/1409)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
Meant to do that a few days ago — the HB that is, not tagging something in KoboUSBMS :-P

**Comment by Frenzie** [comment]
Before you merge, looks like the Android builds failed.

**Comment by NiLuJe** [comment]
Looks like yet another CircleCI brainfart:

```
/bin/sh: 1:  && sed -i -e "s|DIST_SUBDIRS = libdjvu.*|DIST_SUBDIRS = libdjvu|g" Makefile && sed -i -e "s|SUBDIRS = libdjvu tools share \$(am__append_1) \$(am__append_2)|SUBDIRS = libdjvu share \$(am__append_1) \$(am__append_2)|g" Makefile && sh /home/ko/base/thirdparty/djvulibre/../cmake_modules/patch-wrapper.sh /home/ko/base/thirdparty/djvulibre/android.patch: not found
ninja: build stopped: subcommand failed.
Makefile.third:163: recipe for target 'build/arm-linux-androideabi/libs/libdjvulibre.so.21' failed
```

That:

* Doesn't make any sense
* Is completely unrelated to this PR

:?

**Comment by Frenzie** [comment]
Is it my fault? https://github.com/koreader/koreader-base/pull/1404

**Comment by Frenzie** [comment]
On phone, bit hard to check

File: thirdparty/djvulibre/CMakeLists.txt [hunk 1]
```diff
@@ -34,6 +34,8 @@ endif()
 
 if(NOT $ENV{WIN32})
     set(PATCH_CMD sh -c "${ISED} \"s|-lstdc++||g\" libtool")
+else()
+    set(PATCH_CMD sh -c "true")
```
Comment (Frenzie) [inline thirdparty/djvulibre/CMakeLists.txt:38]: Seems to work, but I think it'd be easier to follow if you simply defined it first.

I.e., something like:

```
# Apparently CMake is scoped.
set(PATCH_CMD sh -c "true")

if(NOT $ENV{WIN32})
    set(PATCH_CMD sh -c "${ISED} \"s|-lstdc++||g\" libtool")
```

File: thirdparty/djvulibre/CMakeLists.txt [hunk 2]
```diff
@@ -32,6 +32,8 @@ if($ENV{ANDROID})
     set(CFG_CMD "${CFG_CMD} && ${ISED} 's|soname_spec=.*|soname_spec=\"\\\\\$libname\\\\\$release\\\\\$shared_ext\\\\\$major\"|' libtool")
 endif()
 
+# Apparently CMake is scoped.
```
Comment (Frenzie) [inline thirdparty/djvulibre/CMakeLists.txt:35]: I don't think what I wrote late last night makes sense given that `else` is just as scoped as `if`. :-P

I don't get it though; $ENV{ANDROID} should always combine with NOT $ENV{WIN32}.

**Pull request merged**